### DISCUSSION TITLE
1970s/1970/pdp7unix/simh/

### DISCUSSION THREAD

**Discussion by giscus** [InstallerLegacy/virtualhub.eu.org#2] [Comments]
# 1970s/1970/pdp7unix/simh/

We can run ! PDP7 Unix on the SIMH PDP-7 emulator. First, we need to download the ! PDP7 Unix kit. You can download the kit needed to run it on the SIMH PDP-7 emulator from our GitHub repo.

https://virtualhub.eu.org/1970s/1970/pdp7unix/simh/

**Comment by pulkitkrishna00** [comment]
`dmr` stands for “Dennis MacAlistair Ritchie”, while `ken` stands for “Kenneth Lane Thompson”. They were the original developers of Unix. Dennis Ritchie was also the main developer of the C programming language (He is the R in K&R C).
